docs/troubleshooting/troubleshooting.md

```diff
@@ -0,0 +1,112 @@
+# Troubleshooting
```

Comment:
We should rename this. Otherwise the "Troubleshooting" page (the file above this one) will have a subpage called "Troubleshooting" (this file).

Or, even better: how about we break this file into several pages and link them directly in the index file above?

Reply:
I like this idea. Initially, the pages that are spawned from this are going to be pretty short, which shouldn't be a problem. Long term, we'll probably want to organize thematically. 

Playing devil's advocate, by doing the user does lose the ability to ctrl+f on the page to find a term or topic. We rely on them to navigate to the appropriate topic first.

What do you think?

Reply:
If you can come up with an obvious theme system that will get people to click into the right one then go ahead and break. Otherwise if it's not too much content right now we could leave it all in one and just rename the second "Troubleshooting" so that it is a title specific to that content.

Reply:
Renamed to "Common Issues"